src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const current = { ...state };
+  const result = [];
```

Comment:
You should keep your namings maximally clear. Neither `current` nor `result` gives info about what is stored inside.
```suggestion
  const currentVersion = { ...state };
  const stateVersions = [];
```